.eslintrc

```diff
@@ -43,8 +43,8 @@
         "consistent-return": [ 2 ],
         "curly": [ 2 ],
         "default-case": [ 2 ],
-        "dot-notation": [ [ 2 ], { "allowKeywords": true } ],
-        "eqeqeq": [ 2 ],
+        "dot-notation": [ 2, { "allowKeywords": true } ],
+        "eqeqeq": [ 2, "smart" ],
```

Comment:
I'd like to avoid the work of sorting out which of the various `!= null` comparisons need to also take account of undefined.